### PR TITLE
Update title of controlled section in Breadcrumb demo page (#5653)

### LINKED ISSUE #5653: Update the title of the `Controlled BitBreadcrumb` section of the `Breadcrumb` demo page
The current title of this section is too long and not appropriate:

image

This title should be changed to `Events`.

The demo file name is `BitBreadcrumbDemo.razor` in the `Bit.BlazorUI.Demo.Client.Core` project.

**Note**: There are three different locations for this change on this demo page.

### PR DESCRIPTION
**Pull request by hootanht**
closes #5653

### CODE REVIEW

File: src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Navs/Breadcrumb/BitBreadcrumbDemo.razor.scss [hunk 1]
```diff
@@ -33,4 +33,4 @@
         max-width: 300px;
         flex-flow: column nowrap;
     }
-}
\ No newline at end of file
+}
```
Comment (msynk) [inline src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Navs/Breadcrumb/BitBreadcrumbDemo.razor.scss:36]: revert this change plz

Comment (hootanht) [reply src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Navs/Breadcrumb/BitBreadcrumbDemo.razor.scss]: Fix that.

**Review by msynk — APPROVED** [review]
